src/MaterialDesignThemes.Wpf/Converters/ExpanderRotateAngleConverter.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace MaterialDesignThemes.Wpf.Converters;
 
-internal class ExpanderRotateAngleConverter : IValueConverter
+public class ExpanderRotateAngleConverter : IValueConverter
```

Comment:
I propose moving this inside of a new `MaterialDesignThemes.Wpf.Converters.Internal` namespace.